library/std/src/keyword_docs.rs

```diff
@@ -707,8 +707,8 @@ mod impl_keyword {}
 ///
```

Comment:
Just above this line, like ```**`in`**```.

Reply:
This style doesn't seem to be used anywhere in this file at least.
I also think removing the bold is enough.